solr/solr-ref-guide/modules/upgrade-notes/pages/major-changes-in-solr-9.adoc

```diff
@@ -104,6 +104,44 @@ All the usages are replaced by BaseHttpSolrClient.RemoteSolrException and BaseHt
 In previous versions there was a field returned in async backup status responses, `Response`. This has now been renamed to `msg`, to better fit other collections API responses.
 The `response` field is now a map, containing information about the backup (`startTime`, `indexSizeMB`, `indexFileCount`, etc.).
 
+* SOLR-15982: For collection's snapshot backup request responses additional fields `indexVersion`, `indexFileCount`, etc. were added similar to incremental backup request responses. Also, both snapshot and incremental backup request responses will now contain `starTime` and `endTime`. Here is an example of how collection's snapshot backup request responses will looks like:
+
+Incremental:
+
+```json
+"response": {
+    "collection": "techproducts",
+    "numShards": 2,
+    "backupId": 0,
+    "indexVersion": "9.0.0",
+    "startTime": "2022-02-11T17:20:44.157305500Z",
+    "indexFileCount": 22,
+    "uploadedIndexFileCount": 22,
+    "indexSizeMB": 0.007,
+    "uploadedIndexFileMB": 0.007,
+    "shardBackupIds": [
+        "md_shard2_0",
+        "md_shard1_0"
+    ],
+    "endTime": "2022-02-11T17:20:45.245534400Z"
+}
+```
+
+Snapshot:
+
+```json
+"response": {
+    "collection": "techproducts",
+    "numShards": 2,
+    "indexVersion": "9.0.0",
+    "startTime": "2022-02-11T17:19:33.271461700Z",
+    "indexFileCount": 22,
+    "endTime": "2022-02-11T17:19:34.363859100Z"
+}
+```
```

Comment:
Suggest to omit the example here and to update and/or extend the "Output Snippet" in https://github.com/apache/solr/blob/main/solr/solr-ref-guide/modules/deployment-guide/pages/backup-restore.adoc#backup-status instead.

Reply:
> Suggest to omit the example here and to update and/or extend the "Output Snippet" in https://github.com/apache/solr/blob/main/solr/solr-ref-guide/modules/deployment-guide/pages/backup-restore.adoc#backup-status instead.

I'm glad to, but it seems this is a page about Core API operations, not Collection API operations. I've checked the page for Collection API, but they doesn't have any examples of responses, so nothing to update there. If you know other places in documentation please let me know.

Concerning backup-status document, I will check the shard status response, if it has changed, I will update the document...

Reply:
I agree that this is too much for the major changes page. I think you can add it to the collections api entry if you want, and style it like this: https://solr.apache.org/guide/8_11/collections-api.html#examples-of-async-requests, but it's not a requirement for this PR.